setup.py

```diff
@@ -64,7 +64,20 @@
         "teachable": ["chromadb"],
         "lmm": ["replicate", "pillow"],
         "graph": ["networkx", "matplotlib"],
-        "websurfer": ["beautifulsoup4", "markdownify", "pdfminer.six", "pathvalidate"],
+        "websurfer": [
```

Comment:
Optional: document in https://microsoft.github.io/autogen/docs/installation/Optional-Dependencies
Only if there is a plan to maintain this dependency long-term.